src/containers/shared/components/TransactionActionIcon/TransactionActionIcon.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { TransactionAction } from '../Transaction/types'
+import { getAction } from '../Transaction'
+import { ReactComponent as TransactionCancelIcon } from './TransactionCancelIcon.svg'
+import { ReactComponent as TransactionCreateIcon } from './TransactionCreateIcon.svg'
+import { ReactComponent as TransactionFinishIcon } from './TransactionFinishIcon.svg'
+import { ReactComponent as TransactionModifyIcon } from './TransactionModifyIcon.svg'
+import { ReactComponent as TransactionSendIcon } from './TransactionSendIcon.svg'
+import { ReactComponent as TransactionUnknownIcon } from './TransactionUnknownIcon.svg'
+
+export interface TransactionActionIconProps {
+  action?: TransactionAction
+  type?: string
+}
+
+export const TransactionActionIcon = ({
+  action,
+  type,
+}: TransactionActionIconProps) => {
+  const icons: Record<string, any> = {
```

Comment:
Is there an actual type to use instead of `any`? Isn't there a React component type?